input/assets/js/main.js

```diff
@@ -149,6 +149,7 @@ $("ul#bzoom").each( function(index, ul) {
 // Toggle hidden/sliced description
 $(function() {
     var hiddenDescription = $('p.pdp-product-description');
+    var generatedHidden, hiddenFlag;
 
     hiddenDescription.each(function(){
```

Comment:
@adnasa We could remove the `each`, there is only one instance of this on the page